recipes/hickle/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hickle" %}
+{% set version = "3.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cc4d689129493a59c7872806f69380d1e70d497b9911b7d7a68cd2bd18c43186
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - h5py
+
+test:
+  requires:
+    - pytest
+  imports:
+    - hickle
+
+about:
+  home: http://github.com/telegraphic/hickle
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A HDF5-based python pickle replacement for dumping and loading python variables to file.'
+
+  # The remaining entries in this section are optional, but recommended
```

Comment:
Remove this comment.

Reply:
Ah, sorry I missed that, I've removed that line in my latest commit